lib/spec/simpleofflinecache.sw-spec.js

```diff
@@ -101,4 +104,36 @@ describe('SimpleOfflineCache instance', function () {
     });
   });
 
+  if (!isChrome) {
+    it('should respect ignoreSearch when matching', function(done) {
```

Comment:
I'm not sure about this test. We are trying to test if given some match `options` `.match()` is working as expected but this is something that browser already do during their tests. I think we should test if `.match()` is properly called.